core/src/main/scala/org/apache/spark/util/Benchmark.scala

```diff
@@ -91,16 +92,31 @@ private[spark] object Benchmark {
    * This should return something like "Intel(R) Core(TM) i7-4870HQ CPU @ 2.50GHz"
```

Comment:
nit: Maybe update the comment to to reflect the new format (e.g. "CPU   :Intel(R) Core(TM) i7-4870HQ CPU @ 2.50GHz")